http-add-on/templates/keda-http-operator.yml

```diff
@@ -0,0 +1,204 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    httpscaledobjects.http.keda.sh/version: {{ .Values.version }}
+    keda.sh/addon: keda-http-addon
+  name: keda-http-addon-leader-election-role
+  namespace: {{ .Release.Namespace }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
```

Comment:
Does it need delete permissions?

Reply:
I think it does because it deleted all resources when it's deleted too, but I'm not sure

Reply:
👌